test/e2e/git_test.go

```diff
@@ -63,7 +70,7 @@ directories:
       verification:
         publicKeysSecretRef:
           name: git-pubs
-`, encodedPubKeys, repoPath, ref))
+`, encodedPubKeys, encodedPubKeys, repoPath, ref))
```

Comment:
no need to modify this test since it's not directly related to this feature (and unit tests should give us enough confidence in this case).

Reply:
Removed from e2e